ARROW-2206: [JS] Document Perspective project

lmeyerov opened the pull request:
See https://issues.apache.org/jira/browse/ARROW-2206

+ https://github.com/deepankarsharma

wesm commented on js/README.md:
```diff
@@ -212,6 +212,7 @@ Full list of broader Apache Arrow [projects & organizations](https://github.com/
 
 * [Apache Arrow](https://arrow.apache.org) -- Parent project for Powering Columnar In-Memory Analytics, including affiliated open source projects
 * [rxjs-mapd](https://github.com/graphistry/rxjs-mapd) -- A MapD Core node-driver that returns query results as Arrow columns
+* [Perspective](https://github.com/jpmorganchase/perspective) -- Spinning out of JP Morgan Chase, Perspective is a streaming data visualization engine for Javascript that makes it simple to build real-time & user configurable analytics entirely in the browser.
```
I think it would be OK to have this as an example use case for the JS library (vs. solely on Powered By), minus the editorial langage (e.g. "makes it simple" is subjective). What does "spinning out" mean here? We should try to limit name dropping companies (e.g. you won't see Cloudera mentioned in the README for Apache Kudu or Apache Impala, even though most of the devs works there) outside of Powered By

wesm commented on site/powered_by.md:
```diff
@@ -91,6 +91,7 @@ short description of your use case. See the following for some examples.
 * **[pandas][12]:** data analysis toolkit for Python programmers. pandas
   supports reading and writing Parquet files using pyarrow. Several pandas
   core developers are also contributors to Apache Arrow.
+* **[Perspective][23]:** Perspective is a streaming data visualization engine for Javascript that makes it simple to build real-time & user configurable analytics entirely in the browser. It spun out of JP Morgan Chase.
```
What does "spun out of" mean here? Maybe "created at"?

lmeyerov commented:
-- I removed JP Morgan stewardship from main arrow docs, though still seems relevant for JS powered by so kept there
-- I removed superlatives

I didn't take action on adding to 'Examples' section. I think we can do a lot better on the arrow js intro section, and recommend approaching that more holistically, vs. via this...
